constellations/gemini-admin/src/classes/classEdit.js

```diff
@@ -102,6 +123,14 @@ export class ClassEditPage extends React.Component {
         this.setState({ [value]: event.target.value });
     };
 
+    onChangeSelect = (e) => {
+        const value = e.target.value;
+        console.log(value);
```

Comment:
when you've confirmed things work, please remove "debugging" print statements. :)